modules/server/src/main/scala/observe/server/Systems.scala

```diff
@@ -384,7 +383,7 @@ object Systems {
   }
 
   private def decodeTops(s: String): Map[String, String] =
-    s.split("=|,")
+    s.split("[=,]")
```

Comment:
Is this related to xjc change?

Reply:
No, it was just a suggestion from the IDE. The expressions are equivalent.